NuGet.config

```diff
@@ -9,6 +9,8 @@
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="arcade-validation" value="https://dotnetfeed.blob.core.windows.net/arcade-validation/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="arcade-validation" value="https://dotnet.myget.org/F/symreader-converter/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
nuget.org is already there, and arcade-validation is a key already

Reply:
It also looks like we're missing  https://dotnet.myget.org/F/symreader/api/v3/index.json

Reply:
Oh yeah, I was in the middle of the update when you dropped by and I almost got a heart attack :)

Reply:
Always Ricardo's fault 😭